### ISSUE TITLE
Does this still work?

### ISSUE THREAD

**Issue by sinrise** [jonathantneal/flexibility#78]
I've followed all instructions but so far this has had utterly no effect on my broken site in IE. Looks perfect everywhere else, including Edge, but in any version of IE the layout is broken. I've added the -js-display to my css, included the script, and the 'supportsFlexbox' function, and included the data-style attribute to my code. There are no JS errors but it's still just not working.

**Comment by jasonmonsef** [comment]
The last 1.0 version works for me, the 2.0 version has not worked for me no matter what i try :/ Stoked to at least be able to offer partial support for IE users, cuz i was about to ditch them entirely because of how much i love flex box ;)

**Comment by djmtype** [comment]
Nope, doesn't seem to work in IE9. The demo in IE9 doesn't seem to work either. Is there a link to version 1 somewhere? Version 2.0 is the earliest version in the history.

**Comment by imshuffling** [comment]
Same here - Version 1.06 works for me, https://github.com/jonathantneal/flexibility/releases

**Comment by liu-dongyu** [comment]
@imshuffling How to use flexibility in v1.0.6 ? `flexibility(document.documentElement);` seems not work;

**Comment by imshuffling** [comment]
@liu-dongyu I just used version 1 and referenced flexbox in the stylesheet and it worked for me. What are you trying to achieve?

**Comment by swillis93** [comment]
I'm using the v1.0.6 file, post css module to prefix all of my css properties when compiling, and then I have 'flexibility(document.body);' in my site.js file. However, I'm just getting the following error:
'Uncaught TypeError: flexibility is not a function'

Is anyone able to help?

**Comment by bojandragojevic** [comment]
Same here, demo is not working. Tried to test it in saucelabs ie9 on windows7 but doesn't work.

**Comment by AdamMarsden** [comment]
I can't even get the older 1v to work any ideas what's going on?

**Comment by vlabs** [comment]
:+1:

**Comment by fredbutters** [comment]
I got version `1.0.6` to work so long as I add the prefix:  `-js-flex` or use it  with the `postcss-flexibility` plugin. It's working on a Win7/IE9 Virtual machine at least

**Comment by danyj** [comment]
Same here , no effect , ie11 emulation

**Comment by AurelioDeRosa** [comment]
I've tried to use this polyfill with Bootstrap 4 but no success. It'd be nice to have an official word about it since it seems this is a common problem. Thank you for the effort.

**Comment by sunshine824** [comment]
Where can I download 1.0.6?

**Comment by swillis93** [comment]
@sunshine824

https://github.com/jonathantneal/flexibility/releases/tag/v1.0.6

**Comment by lucas-medina** [comment]
Hey, guys. Version 2.0.1 works on IE9 when embedding the script and using the function `flexibility(document.documentElement)`, as stated on README.md. 

Can you check it out? :) it works for me

**Comment by 823844596** [comment]
Is there anyone solve this problem?
